py/parse_and_notify.py

```diff
@@ -93,7 +93,6 @@ def build(cls, raw_content: Mapping[str, Any]) -> UpdatedTopic:
                 raw_content["numPartitions"],
                 raw_content["numPartitions"],
             ],
-            ["Replication Factor", "UNSUPPORTED", "UNSUPPORTED"],
```

Comment:
removed because topicctl (currently) can't update replication factor so this won't be implemented